src/clarity-angular/wizard/wizard.ts

```diff
@@ -162,12 +163,17 @@ export class Wizard implements OnInit, OnDestroy, AfterContentInit, DoCheck {
     }
 
     public ngAfterContentInit() {
+        let navService = this.navService;
+
         this.pageCollection.pages = this.pages;
-        this.navService.wizardHasAltCancel = this.stopCancel;
-        this.navService.wizardHasAltNext = this.stopNext;
         this.headerActionService.wizardHeaderActions = this.headerActions;
+
+        navService.wizardHasAltCancel = this.stopCancel;
+        navService.wizardHasAltNext = this.stopNext;
+        navService.forceForwardNavigation = this.forceForward;
```

Comment:
I just realized, what if these inputs change dynamically at some point? They're not propagated to the navService, so the service will be out of sync with the inputs given by the user.

Why not just use:
```typescript
@Input("clrWizardPreventDefaultCancel") 
set stopCancel(value: boolean) {
    this.navService.wizardHasAltCancel = value;
}
```